### DISCUSSION TITLE
Limit the points on line chart

### DISCUSSION THREAD

**Discussion by binshadpix** [patrykandpatrick/vico#450] [Ideas]
it would be great if you can give an option to limit the points on line chart only at start and end!

**Comment by patrickmichalik** [comment]
Thanks for the suggestion! We’ll be looking into it.

**Comment by patrickmichalik** [comment]
`LineCartesianLayer.PointProvider`, introduced in [Vico 2.0.0 Alpha 23](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.23), makes this possible. Cheers!

**Reply by binshadpix** [reply]
Thanks for considering the request!

**Comment by binshadpix** [comment]
Hi @patrickmichalik,
is this possible in version 1.12.0 ?

**Reply by Gowsky** [reply]
Hello, @binshadpix. No, as @patrickmichalik mentioned, this was introduced in [Vico 2.0.0 Alpha 23](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.23). 1.12.0 is older than Vico 2.0.0 Alpha 23. In fact, it's older than this feature request.

**Reply by binshadpix** [reply]
okay thanks!

**Comment by Gowsky** [comment]
[Vico 2.0.0](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0) is now available.
